.ci/test_r_package_windows.ps1

```diff
@@ -0,0 +1,108 @@
+# Download a file and retry upon failure. This looks like
+# an infinite loop but CI-level timeouts will kill it
+function Download-File-With-Retries {
+  param(
+    [string]$url,
+    [string]$destfile
+  )
+  do {
+    Write-Output "Downloading ${url}"
+    sleep 5;
+    (New-Object System.Net.WebClient).DownloadFile($url, $destfile)
+  } while(!$?);
+}
+
+$env:R_WINDOWS_VERSION = "3.6.3"
```

Comment:
Do you think we should test with R 4.0 ?

Reply:
yes I think we should have a separate issue for testing on R 4.0.0 across all platforms, now that it's officially released. Good call!

I just opened #3024 to capture that work (and added it to  #2302 )